lib/mage/Mage.js

```diff
@@ -222,6 +222,10 @@ Mage.prototype.getTask = function () {
 Mage.prototype.setupCoreLibs = function () {
 	var core = this.core;
 
+	// Load the configuration
+
+	core.config = require('../config');
```

Comment:
https://github.com/mage/mage/pull/115/files#diff-5122ec46a4f7ff0f1909a379117ff89bR6 and some code a bit below should provide config with a default logger. So I *think* this is enough, but let me know if it isn't.